docs/guide/using-your-ide.md

```diff
@@ -258,9 +258,13 @@ file(GLOB C_SOURCES *.c)
 file(GLOB CPP_SOURCES *.cpp)
 set(SOURCES ${C_SOURCES} ${CPP_SOURCES})
 
+# Set the  Webots home path (Changed it based on your installation method)
+set(WEBOTS_HOME "/snap/webots/current/usr/share/webots")
+#set(WEBOTS_HOME "/usr/share/webots")
```

Comment:
I would prefer to document the standard installation path rather than the snap one:
```suggestion
set(WEBOTS_HOME "/usr/local/webots")
#set(WEBOTS_HOME "/snap/webots/current/usr/share/webots")
```

Reply:
Ok, I think most people prefer debs.